src/App/src/Handler/NotFoundHandler.php

```diff
@@ -14,6 +14,6 @@ class NotFoundHandler implements RequestHandlerInterface
 {
     public function handle(ServerRequestInterface $request): ResponseInterface
     {
-        return new EmptyResponse(StatusCodeInterface::STATUS_NOT_FOUND, ['Content-Type' => 'text']);
+        return new EmptyResponse(StatusCodeInterface::STATUS_NOT_FOUND, ['Content-Type' => 'application/json']);
```

Comment:
```suggestion
        return new EmptyResponse(StatusCodeInterface::STATUS_NOT_FOUND, ['Content-Type' => 'text/plain']);
```
Since the response we are returning has no body, we cannot set `Content-Type=application/json` because when the client tries to parse an empty body as JSON, it will fail.